webui/react/src/pages/Settings/UserManagement.tsx

```diff
@@ -116,7 +116,7 @@ const UserActionDropdown = ({ fetchUsers, user, groups, userManagementEnabled }:
           EditUserModal.open();
           break;
         case MenuKey.Groups: {
-          const response = await getGroups({ userId: user.id });
+          const response = await getGroups({ limit: 500, userId: user.id });
```

Comment:
**suggestion:** I think `limit` of `0` should return the full list, but I noticed that in `apiConfig.ts` it's is force to be `10` if limit is `0`.
```
    detApi.Internal.getGroups({
      limit: params.limit || 10,
      offset: params.offset,
      userId: params.userId,
    }),
```
the `|| 10` should just be removed

Reply:
looks like requesting with limit set to 0 returns an API-level error. could change the API but also wondering if maybe the 500 limit should be kept and the better longer-term solution is incremental loading on the front-end.

Reply:
fwiw, in the python SDK we never relay in the "limit of 0 returns full list" feature but treat every paginated endpoint as possibly requiring additional paging effort, since the limit=0 behavior feels easy to accidentally break... as it sounds like it may have broken in this case...